docs/tags/common-schema.md

```diff
@@ -24,6 +24,11 @@ Refer to this list of default fields for Logs Apps. The tags below are applicabl
 | error.type | A reserved field for errors |
 | span.id | Building block of a trace in distributed tracing |
 | trace.id | Building block of a trace in distributed tracing |
+| geoip | An object reserved for location related data |
```

Comment:
Do we need this at all?  Do we document objects or fields?  Doesn't seeing N fields with the same prefix imply "object"?

Reply:
Looked around, we don't add objects. Removed.